[Enhancement][ESS] Add a note in Manage detection alerts -> Edit drop-down filter controls (#5003)
elastic/security-docs: docs/detections/alerts-ui-manage.asciidoc

====
* You can have a maximum of four controls on the Alerts page.
* You can't remove the *Status* control.
* If you make any changes to the controls, you _must_ save the pending changes for them to persist. 

> * Saved changes are stored in your browser's local storage, not your {ref}user-profile.html[user profile]. If you clear your browser's storage or log into your user profile from a different browser, the default controls on the Alerts page are reapplied.

Reviewer: > If you clear your browser's storage or log into your user profile from a different browser, the default controls on the Alerts page are reapplied.

Somehow word `reapplied` seems not correct. How about below?

If you clear your browser's storage or log into your user profile from a different browser, you will lose your customizations and see default Controls.

Author: I think we can just go with "you'll lose your customizations". The part about your controls reverting to their default state is implied.

Reviewer: Works for me. Thank you.

Author: ```suggestion
* Saved changes are stored in your browser's local storage, not your {ref}user-profile.html[user profile]. If you clear your browser's storage or log into your user profile from a different browser, you will lose your customizations.
```
